frontend/javascripts/navbar.tsx

```diff
@@ -417,11 +417,11 @@ function getDashboardSubMenu(collapse: boolean): SubMenuType {
     label: getCollapsibleMenuTitle("Dashboard", <HomeOutlined />, collapse),
     children: [
       { key: "/dashboard/datasets", label: <Link to="/dashboard/datasets">Datasets</Link> },
-      { key: "/dashboard/tasks", label: <Link to="/dashboard/tasks">Tasks</Link> },
       {
         key: "/dashboard/annotations",
         label: <Link to="/dashboard/annotations">Annotations</Link>,
       },
+      { key: "/dashboard/tasks", label: <Link to="/dashboard/tasks">Tasks</Link> },
```

Comment:
Somewhat unrelated to this PR. At some point in the past we re-ordered the dashboard tabs to make annotations more prominent. Now the navbar items were out of order.